cobalt/browser/user_agent_platform_info.h

```diff
@@ -135,7 +140,8 @@ class UserAgentPlatformInfo : public web::UserAgentPlatformInfo {
   std::string evergreen_type_;
   std::string evergreen_file_type_;
   std::string evergreen_version_;
-  std::string android_os_experience_;  // Only via Client Hints
+  std::string android_build_fingerprint_;  // Only via Client Hints
+  std::string android_os_experience_;      // Only via Client Hints
```

Comment:
The Cobalt extension is already generically named "os experience". Why not keep it generic in the Cobalt common code?